src/lib/machines/probeHybridisation/probeHybridisationMachine.ts

```diff
@@ -215,7 +216,7 @@ export const probeHybridisationMachine = createMachine<ProbeHybridisationContext
         if (e.type !== 'ADD_PROBE_LOT') return;
         const probeLabware = ctx.probeLabware.find((plw) => plw.barcode === e.barcode);
         if (!probeLabware) return;
-        probeLabware.probes.push({ name: '', lot: '', plex: -1 });
+        probeLabware.probes.push({ name: '', lot: '', plex: -1, costing: SlideCosting.Sgp });
```

Comment:
Same here as well, costing field should be empty initially

Reply:
This is due to the type restriction,  I can not set the costing the empty/null as it is of tyoe SlideCosting, on the ui when the user adds a new probe row, the select box is set to empty

Reply:
Why can't the type be defined as SlideCosting | string or SlideCosting|undefined?